src/pkg/backup/details/details.go

```diff
@@ -22,7 +22,7 @@ import (
 
 // Max number of items for which we will print details. If there are
 // more than this, then we just show a summary.
-const maxPrintLimit = 15
+const maxPrintLimit = 100
```

Comment:
100 seems a bit much. 15 might be small, but I feel like with 100, the information will be too much to be useful as console output.

Reply:
reduced to 50